java/test/jmri/jmrix/cmri/serial/cmrinetmetrics/CMRInetMetricsDataTest.java

```diff
@@ -0,0 +1,37 @@
+package jmri.jmrix.cmri.serial.cmrinetmetrics;
+
+import apps.tests.Log4JFixture;
+import jmri.util.JUnitUtil;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Test;
+import java.awt.GraphicsEnvironment;
+
+/**
+ * Test simple functioning of CMRInetMetricsData
+ *
+ * @author	Chuck Catania Copyright (C) 2017
+ */
+public class CMRInetMetricsDataTest {
+
+    @Test
+    public void testMemoCtor() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        CMRInetMetricsDataTest action = new CMRInetMetricsDataTest(); 
```

Comment:
Is the right class being created here? (This might cause CI to fail with the CMRInetMetricsData showing no tests)

Reply:
I am not sure what would be the appropriate tests for the MetricsCollector as it is started and runs headless, listening to the CMRInet traffic.  The MetricsData class is a data structure and access methods for the collected data.
I need some direction on how to approach the tests.